test/evergreen.yml

```diff
@@ -435,20 +435,20 @@ functions:
   "upload artifact":
     - command: archive.targz_pack
       params:
-        target: "wiredtiger.tgz"
-        source_dir: "wiredtiger"
+        target: ${upload_filename|wiredtiger.tgz}
+        source_dir: ${upload_source_dir|wiredtiger}
         include:
           - "./**"
     - command: s3.put
       params:
         aws_secret: ${aws_secret}
         aws_key: ${aws_key}
-        local_file: wiredtiger.tgz
+        local_file: ${upload_filename|wiredtiger.tgz}
         bucket: build_external
         permissions: public-read
         content_type: application/tar
         display_name: Artifacts
-        remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${task_name}_${build_id}${postfix|}.tgz
+        remote_file: ${upload_dest_dir|wiredtiger}/${build_variant}/${revision}/artifacts/${task_name}_${build_id}${postfix|}.tgz
```

Comment:
Is there any special consideration to avoid using the default "wiredtiger" folder to store the tarball on s3?

Reply:
Not really, I was simply willing to separate things since it's mongo-tests and not WiredTiger. Should we keep `wiredtiger` ?

Reply:
Alright got it. I prefer using 'wiredtiger' as it's part of the wiredtiger Evergreen tests.